### ISSUE TITLE
Add a /commands folder and put the commands inside that, and add a /events folder and events inside that.

### ISSUE THREAD

**Issue by ghost** [Prodigy-Hacking/Discord-Github-Database#1]
(not sure if this is a to-do, but suggesting it in case it's not)

This could help improve the simplicity of the code. An example of this could be seen in my bot: [Prodigy-Legends/Prodigy-Legends-Moderation](https://github.com/Prodigy-Legends/Prodigy-Legends-Moderation)

**Comment by ArcerionDev** [comment]
yeah, I was considering doing that, but the bot doesn't have too many commands, so idk if it's necessary. I kinda suck at this kind of thing, so I only have a basic understanding of how to do that, but I may look into it.

**Comment by ghost** [comment]
Okay. I read a tutorial called something like "make a Discord bot in 15 minutes" and that showed me how to create the /commands and /events thing. You may want to look into that.

**Comment by ArcerionDev** [comment]
Alright, thanks
